test/subspace-test-service/src/mock.rs

```diff
@@ -517,21 +608,82 @@ fn log_new_block(block: &Block, used_time_ms: u128) {
     );
 }
 
+struct GossipWorkerPreliminary {
+    gossip_msg_sink: TracingUnboundedSender<Message>,
+    gossip_msg_stream: TracingUnboundedReceiver<Message>,
+    domain_tx_pool_sinks: BTreeMap<DomainId, DomainTxPoolSink>,
+}
+
+impl Default for GossipWorkerPreliminary {
+    fn default() -> Self {
```

Comment:
Using `new()` is semantically better for me.

Reply:
Got a complaint from clippy https://rust-lang.github.io/rust-clippy/master/index.html#new_without_default.